android/java/org/chromium/chrome/browser/preferences/BravePreferenceKeys.java

```diff
@@ -25,6 +25,11 @@ public final class BravePreferenceKeys {
             "org.chromium.chrome.browser.upgrade.Milliseconds_New";
     public static final String BRAVE_DOWNLOADS_AUTOMATICALLY_OPEN_WHEN_POSSIBLE =
             "org.chromium.chrome.browser.downloads.Automatically_Open_When_Possible";
+
+    // Value is being used in
+    // browser/download/internal/android/java/src/org/chromium/chrome/browser/download/BraveDownloadMessageUiControllerImpl.java
```

Comment:
We should move `BravePreferenceKeys.java` to base to be accessible in all components, like we did with `BraveFeatureList.java`